src/App.tsx

```diff
@@ -15,36 +22,118 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+function getPreparedGoods(
+  goods: string[],
+  options: { sortField: SortType | ''; isReversed: boolean },
+): string[] {
+  const prepearedSortGoods = [...goods];
```

Comment:
There's a typo in the variable name `prepearedSortGoods`. It should be `preparedSortGoods` to maintain consistency and clarity in your code.